docs/examples/hpo_launcher.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -v
+
+# Usage:
+#
+#   sbatch --array=0-100 --gres=gpu:1 --cpus-per-gpu=2 --mem-per-gpu=16G scripts/hpo_launcher.sh train.py
+#
+
+# Slurm configuration
+# ===================
+
+#SBATCH --exclude=kepler4,kepler3
+
+# Python
+# ===================
+
+conda activate py39
+
+# Environment
+# ===================
+
+export EXPERIMENT_NAME="MyExperiment"
+
+# Constant
+export SCRATCH=~/scratch
```

Comment:
`$SCRATCH` is now defined in the Mila cluster.

Reply:
fixed